packages/@sanity/cli/src/actions/init-project/templates/nextjs/index.ts

```diff
@@ -12,17 +12,18 @@ import {structureTool} from 'sanity/structure'
 
 // Go to https://www.sanity.io/docs/api-versioning to learn how API versioning works
 import {apiVersion, dataset, projectId} from ${hasSrcFolder ? "'./src/sanity/env'" : "'./sanity/env'"}
-import {schema} from ${hasSrcFolder ? "'./src/sanity/schema'" : "'./sanity/schema'"}
+import {schema} from ${hasSrcFolder ? "'./src/sanity/schemaTypes'" : "'./sanity/schemaTypes'"}
+import {structure} from ${hasSrcFolder ? "'./src/sanity/structure'" : "'./sanity/structure'"}
```

Comment:
Small note 

`import {structure} from './sanity/structure'` the import looks very similar `sanity/structure` would be it be better to name the file `structureLayout` or something along the lines of that.

Reply:
I get that, but also like to name my files and folders the name of the thing it's being imported into, so it's a bit unavoidable. Going to leave for now but thanks!